Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -236,8 +236,29 @@ def self.check_http_content(url, name, user_agents: [:default])
       details[:content_length] == secure_details[:content_length]
     file_match = details[:file_hash] == secure_details[:file_hash]
 
-    return if !etag_match && !content_length_match && !file_match
-    "The URL #{url} could use HTTPS rather than HTTP"
+    if etag_match || content_length_match || file_match
+      return "The URL #{url} could use HTTPS rather than HTTP"
+    end
+
+    return if type != "homepage"
+
+    details[:file] = details[:file].gsub(%r{https?:\\?\/\\?\/}, "/")
```

Comment:
Use a new variable for this e.g. `no_protocol_file_contents =`. The `\/` should not be needed in a `%r`. What's the `\\?`; sorry, being stupid here.

Reply:
`\\?` is to catch protocol prefixes in embedded JavaScript code, escaped as: `https:\/\/`.